docs/user-manual/en/updating-artemis.md

```diff
@@ -0,0 +1,25 @@
+# Updating Artemis
+
+The standard Apache ActiveMQ is runnable out of the box. Just download it, 
+go into the unzipped ActiveMQ folder and run this command: ./bin/activemq run. 
+The ActiveMQ Artemis sub project needs an additional step to run the Message Queue.
+Before running activemq run you have to create a new message broker instance. 
+It looks like an overhead at first glance, but becomes very practically 
+when updating to a new Artemis version for example. 
+To create a artemis broker cd into the artemis folder and run: `./bin/artemis create $HOME/mybroker` on the command line.
+
+> **Note**
+>
+> We recommend choosing a folder different than the downloaded apache-artemis one to separate both from each other.
+> This separation allowes you run multiple brokers with the same artemis runtime for example. 
+> It also simplifies updating to newer versions of Artemis.
+
```

Comment:
Everything in the above paragraphs is already explained in the "Using the Server" chapter of the documentation.